### PR TITLE
Add GitHub Actions, fix formatting, TravisCI, and incompatible Elixir versions

### PR DESCRIPTION
**Pull request by ryanwinchester**
- Added GitHub actions workflow
- Removed Travis CI
- Ran formatter (since CI checks it)
- Bumped Elixir version from v1.8 to v1.9 since `sigil_U` is not available until v1.9 (and it's being used)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,13 @@
 language: elixir
 elixir:
-  - 1.8
+  - 1.9
```
Comment (lpil) [inline .travis.yml]: Let's delete travis as we have github actions now.

**Comment by lpil** [comment]
Thank you. I've left a comment above

**Comment by ryanwinchester** [comment]
@lpil got rid of travis

**Review by lpil — APPROVED** [review]
Thank you!

**Pull request merged**